[DOC] TUI kernel selection (#94)
ROCm/rocm-systems: projects/rocprofiler-compute/docs/how-to/analyze/tui.rst

   $ rocprof-compute analyze --tui

2. To start the individual kernel analysis, use the dropdown menu at the top left of the screen to select
a single workload from ``rocprof-compute profile`` generated output directories.

.. image:: ../../data/analyze/tui_home.png
   :align: center
   :alt: ROCm Compute Profiler TUI home screen
   :width: 800

3. You can see the center window update with a top header for kernel selection and collapsed contents beneath.

> Select a kernel of interest to load the corresponding analysis results, the default kernel selection is the top

Reviewer: ```suggestion
Select a kernel of interest to load the corresponding analysis results. The top kernel is selected by default.
```

> kernel.

Reviewer: ```suggestion
```

.. image:: ../../data/analyze/tui_kernel_selection.png
   :align: center
   :alt: ROCm Compute Profiler TUI home screen
   :width: 800

> 4. After the analysis results are loaded, you can start interactive analysis with detailed metrics by

Reviewer: ```suggestion
4. After the analysis results are loaded, you can start interactive analysis with the detailed metrics by
```

> uncollapse contents to view tables, charts, and graphs visualizing the analysis data.

Reviewer: ```suggestion
expanding the collapsed contents to view tables, charts, and graphs, and visualizing the analysis data.
```

   $ rocprof-compute analyze --tui

> 2. To start the individual kernel analysis, use the dropdown menu at the top left of the screen to select

Reviewer: ```suggestion
2. To start the individual kernel analysis, use the drop-down menu at the top left of the screen to select
```

Unlike the :doc:`cli` plain style interfaces, the TUI restructures the analysis workflow into four
hierarchical categories to provide a more organized, top-down analysis approach:

#. Kernel Selection Header with Top Stats:

   Supports interactive kernel selection to toggle between kernel(s) to view individual kernel
   analysis results.

#. High Level Analysis:

   Experimental performance metrics layout, reorganized performance metrics grouping to display the new
   GPU Speed-of-Light section, Compute Throughput section, and Memory Throughput section.

#. Detailed Block Analysis

   Displays analysis results grouped by metric blocks, similar to the CLI output.

>    When applicable, performance metrics are shown as charts (instead of tables only),

Reviewer: ```suggestion
   When applicable, performance metrics are shown as charts instead of only tables,
```

Unlike the :doc:`cli` plain style interfaces, the TUI restructures the analysis workflow into four
hierarchical categories to provide a more organized, top-down analysis approach:

> #. Kernel Selection Header with Top Stats

Reviewer: ```suggestion
#. Kernel Selection Header with Top Stats:
```

   Supports interactive kernel selection to toggle between kernel(s) to view individual kernel

>    analysis results

Reviewer: ```suggestion
   analysis results.
```

> #. High Level Analysis

Reviewer: ```suggestion
#. High Level Analysis:
```

   Experimental performance metrics layout, reorganized performance metrics grouping to display the new
   GPU Speed-of-Light section, Compute Throughput section, and Memory Throughput section

> #. Detailed Block Analysis

Reviewer: Suggest adding a brief info about "Detailed Block Analysis" similar to other options.

> #. Source Level analysis

Reviewer: Suggest adding a brief info about "Source Level Analysis" similar to other options.
